### PR TITLE
Added new legged robot from Matt Denton

### PR DESCRIPTION
**Pull request by omichel**
Nice looking demo of a Starwars inpired walking robot.

### CODE REVIEW

**Comment by DavidMansolino** [comment]
:eyes:

File: projects/robots/micromagic/scout/worlds/scout.wbt [hunk 1]
```diff
@@ -0,0 +1,840 @@
...
+          ]
+        }
+      ]
+    }
+  ]
+  boundingObject USE BODY_GRP
+  physics Physics {
+  }
+  controller "scout"
+  supervisor TRUE
```
Comment (DavidMansolino) [inline projects/robots/micromagic/scout/worlds/scout.wbt:839]: The supervisor is not used in the controller.

File: projects/robots/micromagic/scout/worlds/scout.wbt [hunk 2]
```diff
@@ -0,0 +1,840 @@
...
+        material Material {
+          diffuseColor 0.454902 0.454902 0.454902
+        }
+      }
+      geometry Plane {
+      }
+    }
+  ]
+  boundingObject USE FLOOR
+}
```
Comment (DavidMansolino) [inline projects/robots/micromagic/scout/worlds/scout.wbt:56]: This floor looks useless.

File: projects/robots/micromagic/scout/worlds/scout.wbt [hunk 3]
```diff
@@ -0,0 +1,840 @@
...
+  direction -0.5 -2 -0.5
+  intensity 0.7
+  castShadows TRUE
+}
+DirectionalLight {
+  ambientIntensity 1
+  direction 0.5 -0.5 0.5
+  intensity 0.7
+  castShadows TRUE
+}
```
Comment (DavidMansolino) [inline projects/robots/micromagic/scout/worlds/scout.wbt:41]: Do we really need 3 DirectionalLight nodes in the scene?

Comment (omichel) [reply projects/robots/micromagic/scout/worlds/scout.wbt:41]: I changed a little bit the lightning so that we now need only 2 directional lights.

Comment (DavidMansolino) [reply projects/robots/micromagic/scout/worlds/scout.wbt:41]: Perfect!

File: projects/robots/Makefile [hunk 4]
```diff
@@ -44,6 +44,7 @@ CONTROLLERS=\
   kuka/youbot/controllers.Makefile \
   lego/mindstorms/controllers.Makefile \
   micromagic/mantis/controllers.Makefile \
+	micromagic/scout/controllers.Makefile \
```
Comment (DavidMansolino) [inline projects/robots/Makefile:47]:
```suggestion
  micromagic/scout/controllers.Makefile \
```

File: projects/robots/micromagic/scout/worlds/scout.wbt [hunk 5]
```diff
@@ -0,0 +1,835 @@
...
+  intensity 0.7
+}
+Floor {
+  translation 0 -0.0001 0
+  size 20 20
+  tileSize 2 2
+  appearance SandyGround {
+  }
+}
+DEF CANVAS Solid {
```
Comment (DavidMansolino) [inline projects/robots/micromagic/scout/worlds/scout.wbt:45]: This second floor is still present.

**Comment by omichel** [comment]
Sorry, that should be all fine now.

**Pull request merged**